crates/nostr/src/event/mod.rs

```diff
@@ -35,7 +35,10 @@ pub enum Error {
     Secp256k1(#[from] secp256k1::Error),
     /// Hex decoding error
     #[error(transparent)]
-    Hex(#[from] bitcoin_hashes::hex::Error),
+    Hex(#[from] bitcoin::hashes::hex::Error),
+    /// OTS error
```

Comment:
Come to think of it, you might want to say "OpenTimestamps error" in the docs, as it's hard to google "OTS"